static/src/stylesheets/components/sass-mq/_mq.scss

```diff
@@ -33,6 +33,7 @@ $mq-responsive: true !default;
 /// @link https://github.com/sass-mq/sass-mq#seeing-the-currently-active-breakpoint Full documentation and examples
 $mq-breakpoints: (
     mobile:  320px,
+    phablet: 620px,
```

Comment:
We tend to always design to the current breakpoint widths, as things can get difficult to understand quickly if the design of your component changes at different times to the rest of the site, in fact looking at the use of the mq mixin, there's only 2 cases of using anything other than the defined widths. What's your reasoning behind needing 620 instead of 660?